app/src/main/kotlin/com/wire/android/ui/markdown/MarkdownComposer.kt

```diff
@@ -233,7 +233,7 @@ fun appendLinksAndMentions(
         append(stringBuilder)
         with(nodeData.colorScheme) {
             linkInfos.forEach {
-                if (it.end - it.start <= 0) {
```

Comment:
suggestion(tests): any test we can add to confirm this works?

Reply:
I will raise another PR with tests for markdowncomposer class